launch/mapper.launch

```diff
@@ -50,8 +46,8 @@
         <param name="fading_memory_update_rate" value="1"/>    <!-- Hz -->
 
         <!-- Map inflation parameters -->
-        <param name="inflate_radius_xy" value="0.30"/>         <!-- meters -->
-        <param name="inflate_radius_z" value="0.30"/>          <!-- meters -->
+        <param name="inflate_radius_xy" value="0.225"/>       <!-- meters -->
+        <param name="inflate_radius_z" value="0.225"/>        <!-- meters -->
```

Comment:
And what's the motivation for this change?

Reply:
This is not a change in this PR, it was a change in https://github.com/Pensasystems/mapper/pull/5